pkgs/by-name/op/openapi-python-client/package.nix

```diff
@@ -0,0 +1,80 @@
+{
+  lib,
+  stdenv,
+  darwin,
+  python3Packages,
+  fetchFromGitHub,
+  installShellFiles,
+  ruff,
+  testers,
+  openapi-python-client,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "openapi-python-client";
+  version = "0.21.4";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    inherit version;
+    owner = "openapi-generators";
+    repo = "openapi-python-client";
+    rev = "v${version}";
+    hash = "sha256-aYwwj96sHfYf3u5Rpcq7QPZA7O4Kt9vVglV38glxuZ0=";
+
+  };
```

Comment:
```suggestion
    hash = "sha256-aYwwj96sHfYf3u5Rpcq7QPZA7O4Kt9vVglV38glxuZ0=";
  };
```

Reply:
Thanks for review, and sorry for pinging you directly but this PR lingered for a bit too long 😅 

Changed this and updated to 0.21.5